libs/ui-components/src/components/ImageBuilds/ImageBuildAndExportStatus.tsx

```
@@ -0,0 +1,136 @@
+  const { t } = useTranslation();
+  if (imageReference) {
+    message = (
+      <Stack hasGutter>
+        <StackItem>{t('Image built successfully')}</StackItem>
+        <StackItem>
+          <ImageUrl imageReference={imageReference} />
+        </StackItem>
+      </Stack>
+    );
+  } else if (message) {
+    message = message;
```

Biome (2.1.2)

[error] message is assigned to itself.

This is where is assigned.

Self assignments have no effect and can be removed.

(lint/correctness/noSelfAssign)